schunk_lwa4d/scripts/move_circ.py

```diff
@@ -0,0 +1,121 @@
+#! /usr/bin/env python
+import math
+import rospy
+
+from geometry_msgs.msg import Pose
```

Comment:
please add the dependency to `geometry_msgs`
https://github.com/ipa320/schunk_robots/blob/indigo_dev/schunk_lwa4d/package.xml